src/components/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div>
+      <Display />
+
+      <div className="btnPanel">
+        <ButtonPanel />
+      </div>
```

Comment:
For clarity and modularity, you could move this to the `ButtonPanel` component.
```suggestion
      <ButtonPanel />
```

Reply:
Thanks, it's implemented on milestone 3.